src/SFML/Window/OSX/HIDInputManager.hpp

```diff
@@ -196,36 +251,48 @@ class HIDInputManager : NonCopyable
     bool isPressed(IOHIDElements& elements);
 
     ////////////////////////////////////////////////////////////
-    /// \brief Convert a HID key usage to its corresponding virtual code
-    ///
-    /// See IOHIDUsageTables.h
+    /// \brief Convert a HID key usage to its corresponding scancode
     ///
     /// \param usage Any kHIDUsage_Keyboard* usage
-    /// \return the virtual code associate with the given HID key usage
-    ///         or 0xff if it is associate with no virtual code
+    /// \return the scancode associate with the given HID key usage
+    ///         or sUnknown if it is associate with no scancode.
```

Comment:
`sUnknown` has changed now, is either `ScanUnknown` or `Scan::Unknown` (depending on how we structure it).

It's also "associated" and not "associate" (2x).